packages/mirinae/src/data-display/field-title/PFieldTitle.vue

```diff
@@ -21,27 +32,54 @@ import { defineProps } from 'vue';
 interface FieldTitleProps {
     label?: string;
     description?: string;
+    fontSize?: 'sm' | 'md' | 'lg';
+    fontWeight?: 'regular' | 'bold';
+    color?: 'dark' | 'gray';
 }
 
 const props = withDefaults(defineProps<FieldTitleProps>(), {
     label: '',
     description: undefined,
+    fontSize: 'md',
```

Comment:
suggestion: How about to use `size` or `titleSize` rather than `fontSize`? Because it's not "font size" 😅 (like 2rem or 16px)